### PR TITLE
RD-6711, RND-209 Seperate environments from services in Blueprints page

### PR DESCRIPTION
**Pull request by PatrycjaKubiczek**
## Description

RD-6711 
RND-209
- Separate environments from services in Blueprints page
- fix bug with creating a new tab container

## Screenshots / Videos

Screenshot 2023-03-16 080237

## Checklist

- [x] My code follows the [UI Code Style](https://cloudifysource.atlassian.net/l/c/x8fq902p).
- [x] I followed the [UI Testing Guidelines](https://cloudifysource.atlassian.net/l/cp/udCVfvrx) and verified that all tests/checks have passed.
- [ ] I verified if that change requires any update in the [documentation](https://cloudifysource.atlassian.net/l/c/4XKtPocy).
- [x] I added proper labels to this PR.

## Tests

 

[Build Status](https://jenkins.cloudify.co/job/Stage-UI-System-Test/4747/) blueprints_spec

### CODE REVIEW

File: app/components/page/content/PageContent.tsx [hunk 1]
```diff
@@ -77,7 +77,10 @@ export default function PageContent({
                                         onLayoutSectionAdded(
                                             {
                                                 type: Consts.LAYOUT_TYPE.TABS,
-                                                content: new Array(2).map(() => ({ name: 'New Tab', widgets: [] }))
+                                                content: map(new Array(2), () => ({
+                                                    name: i18n.t('editMode.tabs.newTab'),
+                                                    widgets: []
+                                                }))
```
Comment (kubama) [inline app/components/page/content/PageContent.tsx:83]: Is there a reason for moving away from `Array.map`?

Comment (PatrycjaKubiczek) [reply app/components/page/content/PageContent.tsx:83]: @kubama I think `new Array ...` created a bug - it creates an array with two empty slots, so in IMO it's not correct 🤔 it also caused the app crashed - the modifed version with `map` was already introduced in onLayoutSectionAdded in another part of this code [L156](https://github.com/cloudify-cosmo/cloudify-stage/blob/37f7be1877de7d755a0974505c6b03d9aef0fc35/app/components/page/content/PageContent.tsx#L157). It creates a an array with two objects containing 'name' and 'widgets' same structure as I saw in [blueprints.json](https://github.com/cloudify-cosmo/cloudify-stage/blob/37f7be1877de7d755a0974505c6b03d9aef0fc35/templates/pages/blueprints.json)

Modified version fixes the issue mentioned here: https://cloudifysource.atlassian.net/browse/RND-209

Comment (kubama) [reply app/components/page/content/PageContent.tsx:83]: How about being just a little bit more explicit and using `_.fill` for this purpose?

Comment (PatrycjaKubiczek) [reply app/components/page/content/PageContent.tsx]: done: https://github.com/cloudify-cosmo/cloudify-stage/pull/2479/commits/f2f3d85cc1c8cbf318ebf80ab579d1590a741ba1 ✔️

**Review by kubama — APPROVED** [review]

**Review by Blewin — APPROVED** [review]
Imo using new Array(2)  instead of just array with two entries is overengineering (that did lead to a bug, from what i hear)

but you fixed the code with minimal changes, so im good either way

**Pull request merged**